src/index.jsx

```diff
@@ -64,6 +64,11 @@ import { Preview } from '@components/preview/index'
 import MintForm from '@components/form/MintForm'
 import { ListsFeed } from '@pages/home/feeds/lists-feed'
 
+if (import.meta.env.BASE_URL !== '/') {
```

Comment:
I'll have to double check but this should theoretically be automatically discarded in the builds of the real deploys